src/components/ContentSidebar/SidebarAccessStats.js

```diff
@@ -35,11 +35,6 @@ const SidebarAccessStats = ({
     intl,
 }: Props) => {
     const { preview_count, comment_count, download_count, edit_count } = accessStats;
-
-    if (!preview_count && !comment_count && !download_count && !edit_count && !error) {
```

Comment:
All of those will be defaulted to 0, in the params list above
```
     accessStats = {	     accessStats = {
         preview_count: 0,	         preview_count: 0,
         comment_count: 0,	         comment_count: 0,
         download_count: 0,	         download_count: 0,
         edit_count: 0,	         edit_count: 0,
         has_count_overflowed: false,	         has_count_overflowed: false,
     },
```

Reply:
@jeremypress's concern is valid. If the value of `accessStats` is null, it will not use the default param; that's only the case for undefined. If `accessStats` _is_ defined and not null and any of its inner values are null or undefined, the default parameter's inner values will not used.

Reply:
@jeremypress @jstoffan The API doesn't return `null` values for access stats, only numbers >= 0. Do we assume here that the API can return null, or follow the API contract?

Reply:
I went ahead and followed the react-intl code and found it uses Intl.NumberFormat under the hood, which seems to format `null` as `"0"` by default. So I guess we should be okay in this case.